source/GlassView.Core/Models/AllocationInfo.cs

```diff
@@ -0,0 +1,9 @@
+namespace GlassView.Core.Models;
+
+public sealed class AllocationInfo
```

Comment:
Should this be a class or a record, as then we have some added properties by default?

Reply:
Sure, I'll change them to record classes.